packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -189,8 +204,14 @@ MeetingUtil.declineMeeting = (meeting, reason) =>
     reason
   });
 
-MeetingUtil.isGuestUnjoined = (locusInfo) =>
-  locusInfo.parsedLocus && locusInfo.parsedLocus.self && locusInfo.parsedLocus.self.guest && locusInfo.parsedLocus.self.state !== _JOINED_;
+MeetingUtil.isUserInLeftState = (locusInfo) =>
+  locusInfo.parsedLocus && locusInfo.parsedLocus.self && locusInfo.parsedLocus.self.state === _LEFT_;
```

Comment:
```suggestion
  locusInfo.parsedLocus?.self?.state === _LEFT_;
```

I know this is a nitpick, but we could clean up these lines with optional chaining